src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -57,6 +57,11 @@ export class SearchMenu extends React.Component {
         sessionStorage.setItem('searchTypeaheadLogged', JSON.stringify(true));
       }
     }
+
+    // focus the query input when the search menu is opened
+    if (isOpen && !prevProps.isOpen) {
+      document.getElementById('query').focus();
```

Comment:
In the case where `document.getElementById('query')` can't find the node, a runtime error will be thrown at `.focus()`, you could prevent this by doing an escape early if statement beforehand

Reply:
fixed